public/admin/config.yml

```diff
@@ -45,26 +45,10 @@ collections:
     extension: "md"
     create: true
     fields:
-      - {
-          label: "Title",
-          name: "title",
-          widget: "string",
-        }
-      - {
-          label: "Date",
-          name: "date",
-          widget: "date",
-        }
-      - {
-          label: "Link Text",
-          name: "link_text",
-          widget: "string",
-        }
-      - {
-          label: "Link",
-          name: "link",
-          widget: "string",
-        }
+      - { label: "Title", name: "title", widget: "string" }
+      - { label: "Date", name: "date", widget: "date" }
+      - { label: "Link Text", name: "link_text", widget: "string" }
+      - { label: "Link", name: "link", widget: "string" }
```

Comment:
eh? Is that from the `informasi-terbaru` collection? I didn't touch any collection other than `emergency-page`. Sorry it could be autoformatted while I merging the branch. I'll fix it. 🙏